tools/bpf2c/bpf2c_driver.c

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#include <guiddef.h>
+#include <wdm.h>
+#include <wsk.h>
+
+#define NO_CRT
+#include "bpf2c.h"
+
+DRIVER_INITIALIZE DriverEntry;
+DRIVER_UNLOAD DriverUnload;
+RTL_QUERY_REGISTRY_ROUTINE static _bpf2c_query_registry_routine;
+
+static GUID _bpf2c_npi_id = {/* c847aac8-a6f2-4b53-aea3-f4a94b9a80cb */
+                             0xc847aac8,
+                             0xa6f2,
+                             0x4b53,
+                             {0xae, 0xa3, 0xf4, 0xa9, 0x4b, 0x9a, 0x80, 0xcb}};
+static NPI_MODULEID _bpf2c_module_id = {sizeof(_bpf2c_module_id), MIT_GUID, {0}};
+static HANDLE _bpf2c_nmr_client_handle;
+static HANDLE _bpf2c_nmr_provider_handle;
+metadata_table_t ___METADATA_TABLE___;
+
+static NTSTATUS
+_bpf2c_npi_client_attach_provider(
+    _In_ HANDLE nmr_binding_handle,
+    _In_ void* client_context,
+    _In_ const NPI_REGISTRATION_INSTANCE* provider_registration_instance);
+
+static NTSTATUS
+_bpf2c_npi_client_detach_provider(_In_ void* ClientBindingContext);
```

Comment:
```suggestion
_bpf2c_npi_client_detach_provider(_In_ void* client_binding_context);
```

Reply:
Unresolving this comment since I don't see the change applied.  The master branch still has this coding convention issue on this line.

Reply:
Darn. I fixed it on line # 137, but missed line 32. Will create another PR to fix this as this one is already merged.